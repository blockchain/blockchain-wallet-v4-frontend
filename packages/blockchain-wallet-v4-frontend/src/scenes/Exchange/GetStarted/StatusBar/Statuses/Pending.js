import React from 'react'
import styled from 'styled-components'
import { FormattedMessage } from 'react-intl'

import { Text } from 'blockchain-info-components'

const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: flex-start;
  width: 100%;
  padding: 16px;
  box-sizing: border-box;
  background-color: #f5a623;
  border-radius: 4px;

  & > :not(:first-child) {
    margin-top: 8px;
  }

  @media (min-width: 960px) {
    width: 500px;
  }
`

export const Pending = () => (
  <Wrapper>
    <Text color='white' size='14px' weight={400}>
      <FormattedMessage
        id='scenes.exchange.getstarted.status.pending.title'
        defaultMessage='Account verification in progress'
      />
    </Text>
    <Text size='14px' weight={300}>
      <FormattedMessage
        id='scenes.exchange.getstarted.status.pending.description'
        defaultMessage='We are currently reviewing your application. Hang tight! In just a few minutes you will be all set to trade cryptocurrency. You should receive an update within 5 minutes.'
      />
    </Text>
  </Wrapper>
)

export default Pending
