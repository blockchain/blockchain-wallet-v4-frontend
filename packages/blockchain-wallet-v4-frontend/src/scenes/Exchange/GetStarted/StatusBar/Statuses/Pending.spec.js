import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Pending } from './Pending'

jest.mock('blockchain-info-components', () => ({
  Text: 'text'
}))

describe('Pending', () => {
  it('renders correctly', () => {
    const component = shallow(<Pending />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
