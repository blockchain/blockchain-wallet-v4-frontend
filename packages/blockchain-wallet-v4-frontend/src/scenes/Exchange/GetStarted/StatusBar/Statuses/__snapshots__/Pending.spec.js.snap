// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pending renders correctly 1`] = `
<Pending__Wrapper>
  <text
    color="white"
    size="14px"
    weight={400}
  >
    <FormattedMessage
      defaultMessage="Account verification in progress"
      id="scenes.exchange.getstarted.status.pending.title"
      values={Object {}}
    />
  </text>
  <text
    size="14px"
    weight={300}
  >
    <FormattedMessage
      defaultMessage="We are currently reviewing your application. Hang tight! In just a few minutes you will be all set to trade cryptocurrency. You should receive an update within 5 minutes."
      id="scenes.exchange.getstarted.status.pending.description"
      values={Object {}}
    />
  </text>
</Pending__Wrapper>
`;
